include/exadg/time_integration/time_int_gen_alpha_base.cpp

```diff
@@ -94,26 +94,34 @@ TimeIntGenAlphaBase<Number>::set_current_time_step_size(double const & time_step
 
 template<typename Number>
 double
-TimeIntGenAlphaBase<Number>::get_scaling_factor_mass() const
+TimeIntGenAlphaBase<Number>::get_scaling_factor_mass_from_acceleration() const
 {
-  return (1.0 - alpha_m) / ((1.0 - alpha_f) * beta * std::pow(time_step, 2.0));
+  return ((1.0 - alpha_m) / ((1.0 - alpha_f) * beta * std::pow(time_step, 2.0)));
+}
+
+template<typename Number>
+double
+TimeIntGenAlphaBase<Number>::get_scaling_factor_mass_from_velocity() const
```

Comment:
I think the time integrator should use the wording `mass_operator` and `damping_operator`. In case the damping operator is very simple, it is the `SpatialOperator` or `ElasticityOperatorBase` (where there is a method called `apply_damping_operator()`) that translates this into a simple mass scaling.

Reply:
It feels like this comment is more general and does not only affect the 3 highlighted lines, does it? If so, we should discuss this in person please, otherwise please consider the following comments:

I think this would mix the time integration constants (which we are actually computing) with the scaling factors in the operators. I would rather focus on the "pure" time integration factors stemming from the different sources here and keep out the physics, otherwise it gets misleading outside in my opinion.

If we take `get_scaling_factor_damping_operator()`, it depends on the actual damping used (strong or weak damping) and one would expect `parameters.weak_damping_coefficient` to contribute. I am in favour of the strict separation of time integration constants and operator scaling, which was prior to this PR not an issue.